config/statusMsg.js

```diff
@@ -0,0 +1,11 @@
+module.exports= {
+    success : {
```

Comment:
As 'success' object contains just one object nested inside, this could be shortened to:
`success: "Success"`